src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSCommand.groovy

```diff
@@ -57,7 +57,13 @@ class PBSCommand extends SubmissionCommand {
 
     @Override
     protected String getEnvironmentExportParameter() {
-        return " -V "
+        if (passLocalEnvironment.isPresent()) {
+            if (passLocalEnvironment.get())
```

Comment:
Elvis like above?

Reply:
if(passLocal... isPresent() && ... .get()) return "-V"
return ""
?

Reply:
This solution is harder to read, because the control flow is broken: The second return does not only require to understand the condition (as in the code that I have), but also what is happening in the if-case. So, even though the alternative is more concise, it is less clear and I would prefer clarity over brevity.